src/awx_plugins/credentials/github_app.py

```diff
@@ -0,0 +1,185 @@
+"""GitHub App Credential Plugin.
+
+This module defines a credential plugin for GitHub Apps, allowing
+authentication via GitHub App tokens.
+
+Functions:
```

Comment:
Broken rendering: https://awx-plugins-core--84.org.readthedocs.build/en/84/pkg/awx_plugins.credentials.github_app/#module-awx_plugins.credentials.github_app.

Perhaps, this will fix it:
```suggestion
Functions:

```